alchemist-web-renderer/src/commonMain/kotlin/it/unibo/alchemist/common/utility/Ruotes.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2010-2023, Danilo Pianini and contributors
+ * listed, for each module, in the respective subproject's build.gradle.kts file.
+ *
+ * This file is part of Alchemist, and is distributed under the terms of the
+ * GNU General Public License, with a linking exception,
+ * as described in the file LICENSE in the Alchemist distribution's top directory.
+ */
+
+package it.unibo.alchemist.common.utility
+
+/**
+ * Objects that store all the route path. Since this object is part of the common sourceSet, both Client and Serve can
+ * refer to the correct Route maintaining consistency and reducing the number of possible errors and bug.
+ */
+object Ruotes {
```

Comment:
```suggestion
object Routes {
```

Reply:
So many places I could have gone wrong and I picked the worst 🤦🏽